src/openloco/windows/options.cpp

```diff
@@ -554,8 +535,9 @@ namespace openloco::ui::options
             }
             w->widgets[widx::screen_mode].text = screen_mode_string_id;
 
-            set_format_arg(0x10, uint16_t, config::get().resolution_width);
-            set_format_arg(0x12, uint16_t, config::get().resolution_height);
+            config::resolution_t resolution = config::get_new().display.fullscreen_resolution;
```

Comment:
Probably should be a reference:
```suggestion
            config::resolution_t& resolution = config::get_new().display.fullscreen_resolution;
```